README.md

```diff
@@ -26,32 +26,29 @@ This implementation shows you how to do the following:
 
-#### Instructions
+### Instructions
 
-First you will login to your AWS account with an Admin user or role. This will allow the successful launch of the CloudFormation stack template. You can deploy the stack and explore our example by following these steps:
+1. Click 'Launch Stack' for the AWS region you want to deploy resources into
 
-To begin, click the Launch link in Step 1 below
+|AWS Region                |     Link        |
+|:------------------------:|:-----------:|
+|us-east-1 (N. Virgnia)    | [<img src="./notebooks/images/cloudformation-launch-stack.png">](https://console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacks/new?stackName=sm-fs-streaming-agg-stack&templateURL=https://s3.amazonaws.com/sagemaker-feature-store-streaming-aggregation-us-east-1/artifacts/latest/sagemaker-featurestore-template.yaml) |
+|us-east-2 (Ohio)          | [<img src="./notebooks/images/cloudformation-launch-stack.png">](https://console.aws.amazon.com/cloudformation/home?region=us-east-2#/stacks/new?stackName=sm-fs-streaming-agg-stack&templateURL=https://s3.amazonaws.com/sagemaker-feature-store-streaming-aggregation-us-east-2/artifacts/latest/sagemaker-featurestore-template.yaml) |
+|us-west-1 (N. California) | [<img src="./notebooks/images/cloudformation-launch-stack.png">](https://console.aws.amazon.com/cloudformation/home?region=us-west-1#/stacks/new?stackName=sm-fs-streaming-agg-stack&templateURL=https://s3.amazonaws.com/sagemaker-feature-store-streaming-aggregation-us-west-1/artifacts/latest/sagemaker-featurestore-template.yaml) |
+|eu-west-1 (Dublin)        | [<img src="./notebooks/images/cloudformation-launch-stack.png">](https://console.aws.amazon.com/cloudformation/home?region=eu-west-1#/stacks/new?stackName=sm-fs-streaming-agg-stack&templateURL=https://s3.amazonaws.com/sagemaker-feature-store-streaming-aggregation-eu-west-1/artifacts/latest/sagemaker-featurestore-template.yaml) |
+|ap-northeast-1 (Tokyo)    | [<img src="./notebooks/images/cloudformation-launch-stack.png">](https://console.aws.amazon.com/cloudformation/home?region=ap-northeast-1#/stacks/new?stackName=sm-fs-streaming-agg-stack&templateURL=https://s3.amazonaws.com/sagemaker-feature-store-streaming-aggregation-ap-northeast-1/artifacts/latest/sagemaker-featurestore-template.yaml) |
 
-1. [Launch](https://us-east-1.console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacks/create/template?stackName=sm-fs-streaming-agg-stack&templateURL=https://aws-ml-blog.s3.us-east-1.amazonaws.com/artifacts/Using-streaming-ingestion-with-Amazon-SageMaker-Feature-Store/sagemaker-featurestore-template.yaml) the AWS CloudFormation stack in us-east-1. 
-To deploy the stack in other regions, [follow these instructions](./create_stack_in_other_regions.md).
+To deploy the stack in other regions, you can [follow these instructions](./create_stack_in_other_regions.md). Please log an issue in this repo if you would like additional regions officially supported.
```

Comment:
You are implying that we will continue to expand the list of regions. Are there any limitations?

Reply:
There aren't any limitations to us supporting additional regions in future if there is interest.